CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 UNRELEASED
 ----------
 
+* [ [#1827](https://github.com/digitalfabrik/integreat-cms/issues/1827) ] Make Multi-language-XLIFF export popup scrollable
+* [ [#1521](https://github.com/digitalfabrik/integreat-cms/issues/1521) ] Show warning when user entered slug got changed
+* [ [#1807](https://github.com/digitalfabrik/integreat-cms/issues/1807) ] Remove short description field from POI form
+* [ [#522](https://github.com/digitalfabrik/integreat-cms/issues/522) ] Add region setting to activate SEO section, add SEO section to POI form
+* [ [#1858](https://github.com/digitalfabrik/integreat-cms/issues/1858) ] Enable submitting feedback about fallback translations of recurring events
+* [ [#1865](https://github.com/digitalfabrik/integreat-cms/issues/1865) ] Fix TextLab HIX widget for non-staff users
+
 
 2022.11.2
 ---------
 
 * [ [#1843](https://github.com/digitalfabrik/integreat-cms/issues/1843) ] Fix region selection after login
+* [ [#1844](https://github.com/digitalfabrik/integreat-cms/issues/1844) ] Use gender sensitive language in user form
```

Comment:
This is weird though.

Reply:
damn, probably I messed up during rebase :D